config/config.go

```diff
@@ -18,6 +18,22 @@ type Config struct {
 	Email      mailutil.Config
 	Database   gormutil.Config
 	Provider   provider.Config
+	Server     Server
+	TLS        TLS
```

Comment:
Could merge the Server and the TLS?

Reply:
I don't think merging Server and TLS is a good idea

Server section can be used to set configs of the service, not only be used for tls